pkgs/development/compilers/yosys/plugins/symbiflow.nix

```diff
@@ -8,25 +8,24 @@
 , yosys
 , zlib
 , yosys-symbiflow
+, uhdm
+, surelog
 }: let
 
   src = fetchFromGitHub {
-    owner  = "SymbiFlow";
-    repo   = "yosys-symbiflow-plugins";
-    rev    = "35c6c33811a8de7c80dff6a7bcf7aa6ec9b21233";
-    hash   = "sha256-g5dX9+R+gWt8e7Bhbbg60O9qa+Vi6Ar0M1sHhYlAre8=";
+    owner  = "chipsalliance";
+    repo   = "yosys-f4pga-plugins";
+    rev    = "aadd1735b2b7af0472e56dc23f1035d6e1904712";
+    hash   = "sha256-gmpEx+8XDN7f+6e/YG25VKgdadwqApPqU3S6OB4AryA=";
   };
 
-  version = "2022.01.06";
+  version = "2022.05.13";
```

Comment:
Maybe it is worthwhile updating this to latest head as well (I guess this PR has sitting for a while)